docker-compose.yml

```diff
@@ -1,150 +1,126 @@
-version: "3"
+x-app: &app
+  build:
+    target: development
+    context: .
+    args:
+      RUBY_VERSION: '3.1.4'
+      PG_MAJOR: '11'
+      NODE_MAJOR: '16'
+      YARN_VERSION: '1.22.18'
+  image: forem-dev:1.0.0
+  environment: &env
+    NODE_ENV: ${NODE_ENV:-development}
+    RAILS_ENV: ${RAILS_ENV:-development}
+  tmpfs:
+    - /tmp
+    - /app/tmp/pids
+
+x-backend: &backend
+  <<: *app
+  stdin_open: true
+  tty: true
+  volumes:
+    - .:/app:cached
+    - bundle:/usr/local/bundle
+    - rails_cache:/app/tmp/cache
+    - assets:/app/public/assets
+    - node_modules:/app/node_modules
+    - packs:/app/public/packs
+    - packs-test:/app/public/packs-test
+    - history:/usr/local/hist
+    - ./.dockerdev/.psqlrc:/root/.psqlrc:ro
+    - ./.dockerdev/.bashrc:/root/.bashrc:ro
+  environment: &backend_environment
+    <<: *env
+    REDIS_URL: redis://redis:6379/
```

Comment:
Our repo defaults to `REDIS_URL="redis://localhost:6379" assuming the engineer is running redis locally. I can't seem to adjust this without it not working though. Is there an obvious answer I'm seeing?